TemporaryProjects.14.0/StartPageExtenderPackage.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading;
+using System.Runtime.InteropServices;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using EnvDTE;
+using Task = System.Threading.Tasks.Task;
+
+namespace TemporaryProjects
+{
+    [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
+    [InstalledProductRegistration("#110", "#112", "1.0", IconResourceID = 400)] // Info on this package for Help/About
+    [Guid(PackageGuids.startPageExtenderPackage14String)]
+    [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1650:ElementDocumentationMustBeSpelledCorrectly", Justification = "pkgdef, VS and vsixmanifest are valid VS terms")]
+    [ProvideAutoLoad(PackageGuids.startPageToolWindowString, PackageAutoLoadFlags.BackgroundLoad)]
+    public sealed class StartPageExtenderPackage : AsyncPackage
```

Comment:
@jcansdale I'm a little confused by your latest changes. Why is this file in TemporaryProjects.14.0?

Reply:
I was hoping there would be a way to only install/load the `StartPageExtenderPackage` package in `TemporaryProjects.15.0` into Visual Studio 2015. Alas this doesn't appear to be possible. 😢 Hopefully I'm missing something and there is a way?

As a workaround the `StartPageExtenderPackage` in `TemporaryProjects.14.0` is loading when the `StartPage` becomes visible. When it detects that it's running inside Visual Studio 2017 (`DTE.Version == "15.0"`), it forces the real `StartPageExtenderPackage` from `TemporaryProjects.15.0` to load.

Does that make sense?

Reply:
Makes sense, but I think you meant to say 2017 instead of 2015 here:
> I was hoping there would be a way to only install/load the StartPageExtenderPackage package in TemporaryProjects.15.0 into Visual Studio 2015.

Reply:
It appears this is no longer true after 8e79128899b821efb460ccb3b795856d328a88a4